stylesheets/_component.modals.scss

```diff
@@ -41,12 +41,12 @@
     margin-left: auto;
     margin-right: auto;
     max-width: 600px;
-    padding: 85px ($gutter-width / 2) 30px;
+    padding: 10px;
```

Comment:
You've used `($gutter-width / 2)` below and `10px` here, explicit choice?

Reply:
Corrected.